app/main.py

```diff
@@ -17,3 +17,18 @@
 collection_of_coins = {1, 2, 25}
 
 # write your code here
+print(f"Name: {name}")
+print(f"Profile Info: {profile_info}")
+
+new_film = "Inception"
+my_favourite_films.append(new_film)
+print("Updated list of favourite films:", my_favourite_films)
+
+total_coin_value = sum(collection_of_coins)
```

Comment:
The calculation of `total_coin_value` using `sum(collection_of_coins)` assumes that the set `collection_of_coins` contains the actual values of the coins. If these are meant to be denominations, and you have multiple coins of each denomination, you should adjust the calculation accordingly. Otherwise, this line is correct if each denomination appears only once.

Reply:
collection_of_coins = [0.25, 0.1, 0.1, 0.05]
total_coin_value = sum(collection_of_coins)
print("Total coin value (list):", total_coin_value)